### PR TITLE
Fix various webapp issues

### PR DESCRIPTION
**Pull request by bourque**
This is a catch-all PR that fixes various issues and makes various enhancements to the web app:

- Updates programmatic API examples to use production server (i.e. `https://jwql.stsci.edu`) instead of dev server
- Adds support for `c1d` filetypes in `utils.filename_parser()`
- Fixes bug in dashboard that was causing webpage to crash in the absence of data
- Removes unnecessary parts of docstrings in `bokeh_dashboard.py`
- Fixes bug in `data_containers.py` that was causing `get_expstart()` to crash on segmentation map files
- Updates the 'Query the archive for anomalies' header on homepage to just 'Query the archive' (since one could query for more than just anomalies)

### CODE REVIEW

**Comment by bourque** [comment]
Areas to check:

- [ ] Homepage
- [ ] About page
- [ ] Archive pages:
    - [ ] NIRCam
    - [ ] NIRISS
    - [ ] NIRSpec
    - [ ] MIRI
    - [ ] FGS
- [ ] Spot check some proposal pages:
    - [ ] NIRCam
    - [ ] NIRISS
    - [ ] NIRSpec
    - [ ] MIRI
    - [ ] FGS
- [ ] Spot check some preview image pages:
    - [ ] NIRCam
    - [ ] NIRISS
    - [ ] NIRSpec
    - [ ] MIRI
    - [ ] FGS
- [ ] Instrument monitors:
    - [ ] NIRCam
    - [ ] NIRISS
    - [ ] NIRSpec
    - [ ] MIRI
    - [ ] FGS
- [ ] Dashboard
- [ ] EDB page
- [ ] API
- [ ] JWQLDB

**Comment by bourque** [comment]
Hey @mfixstsci, I was wondering if you had some time and could review this for me?  If not, no worries, I can probably find someone else.

File: jwql/website/apps/jwql/data_containers.py [hunk 1]
```diff
@@ -460,13 +460,18 @@ def get_expstart(instrument, rootname):
         The exposure start time of the observation (in MJD).
     """
 
-    file_set_name = '_'.join(rootname.split('_')[:-1])
+    if '-seg' in rootname:
```
Comment (mfixstsci) [inline jwql/website/apps/jwql/data_containers.py:463]: Is `-seg` in some kind of engineering or ground test data?

File: jwql/website/apps/jwql/templates/api_landing.html [hunk 2]
```diff
@@ -49,8 +49,8 @@ <h5>Obtaining Filenames Associated with a Rootname</h5>
                     from urllib import request
                     import json
 
-                    # Build the URL https://dljwql.stsci.edu/api/roontname/service/
-                    url = 'https://dljwql.stsci.edu/api/jw93025001001_02102_00001_nrca2/filenames/'
+                    # Build the URL https://jwql.stsci.edu/api/roontname/service/
+                    url = 'https://jwql.stsci.edu/api/jw93025001001_02102_00001_nrca2/filenames/'
```
Comment (mfixstsci) [inline jwql/website/apps/jwql/templates/api_landing.html:53]: Are we moving to the production server now?!

**Review by mfixstsci — COMMENTED** [review]
Everything looks good @bourque just had a couple of small questions!

File: jwql/website/apps/jwql/data_containers.py (hunk 1 above)
Comment (bourque) [reply jwql/website/apps/jwql/data_containers.py:463]: They appear to be some files related to time series observations: https://innerspace.stsci.edu/pages/viewpage.action?pageId=94092600#SummaryofAssociationFileNamingConvention/Design-TSOfiles

File: jwql/website/apps/jwql/templates/api_landing.html (hunk 2 above)
Comment (bourque) [reply jwql/website/apps/jwql/templates/api_landing.html:53]: Yeah, I would like the users to use the production server for this service, so best to use it in the examples.

**Comment by bourque** [comment]
Thanks @mfixstsci!

File: jwql/website/apps/jwql/data_containers.py (hunk 1 above)
Comment (bhilbert4) [reply jwql/website/apps/jwql/data_containers.py:463]: Time series observations that create a data volume above some limit are split into multiple "segment" files. "seg" refers to the segment number. So you can have a single exposure split up into, eg:
jw88888002001_01101_00002-seg001_nrca5_uncal.fits
jw88888002001_01101_00002-seg002_nrca5_uncal.fits
jw88888002001_01101_00002-seg003_nrca5_uncal.fits

Comment (mfixstsci) [reply jwql/website/apps/jwql/data_containers.py:463]: Thanks @bhilbert4!

**Review by mfixstsci — APPROVED** [review]

**Pull request merged**